spec/static/spec/v1/openapitransactional.yaml

```diff
@@ -92,6 +95,34 @@ tags:
         - <span style="color:red">*Files bigger than 10 MB should use compression,
           see [Compression](#section/Compression)*</span>
 
+      ## Steps to upload Assets
+
+      To upload an assets file there are generally three steps involved:
+
+      <ol>
+        <li>Create new upload process</li>
+        <li>Upload parts/files</li>
+        <li>Complete upload process</li>
+      </ol>
+
+      Any file that is larger than 5 GB must be split into multiple parts. A file part must be at
+      least 5 MB except for the last one and at most 5 GB, otherwise the complete operation will fail.
```

Comment:
How about "except for the last (or only) part" ?

Reply:
If we want to call it out, I would add a separate sentence instead of parenthesis. "If there is only one part, it can be smaller than 5 MB since it is also the last part."

Reply:
Following this sentence there is the following sentence:
"If the file is less that 5 GB, you will only upload a single part, but must still start and complete the process as with multiple parts."
Do you think it needs more than this?